package.json

```diff
@@ -29,7 +29,8 @@
     "build": "tsc",
     "build:clean": "shx rm -rf ./dist ./coverage ./.nyc_output",
     "lint": "tslint --project .",
-    "test": "nyc mocha --config .mocharc.json \"src/**/*.spec.ts\"",
+    "mocha": "nyc mocha --config .mocharc.json \"src/**/*.spec.ts\"",
+    "test": "npm run lint && npm run mocha",
```

Comment:
all of the lint errors are fixed now, so this should pass. running as part of test makes sure we don't revert/add new bad things in the future, and get that for free as part of CI.